vsphere/cpi.yml

```diff
@@ -48,14 +48,22 @@
     password: ((vcenter_password))
     datacenters:
     - name: ((vcenter_dc))
-      vm_folder: ((vcenter_vms))
-      template_folder: ((vcenter_templates))
-      datastore_pattern: ((vcenter_ds))
-      persistent_datastore_pattern: ((vcenter_ds))
-      disk_path: ((vcenter_disks))
-      clusters:
-      - ((vcenter_cluster)): {}
+      vm_folder: ((vcenter_vm_folder))
+      template_folder: ((vcenter_templates_folder))
+      datastore_pattern: ((vcenter_ds_ephemeral))
+      persistent_datastore_pattern: ((vcenter_ds_persistent))
+      disk_path: ((vcenter_disk_path))
+      clusters: ((vcenter_cluster))
 
 - type: replace
   path: /cloud_provider/properties/vcenter?
   value: *vcenter
+
+- type: replace
```

Comment:
First one I disagree, as you cannot deploy a bosh director into vsphere with > 1 cluster without exactly specifiying into which cluster the director should go, as bosh create-env will most times screw up the selection of the data store or distributed port group. Second I can agree and i can remove it